dataplane/standalone/translator.h

```diff
@@ -20,14 +20,14 @@
 
 #include <memory>
 
-#include "dataplane/standalone/lucius/lucius_clib.h"
 #include "proto/forwarding/forwarding_service.grpc.pb.h"
 #include "proto/forwarding/forwarding_service.pb.h"
 
 extern "C" {
 #include "inc/sai.h"
 }
 
+
 class Translator {
  public:
   explicit Translator(std::shared_ptr<grpc::Channel> chan) {
```

Comment:
Is the `create_switch` function here still going to be used to populate the tables by overwriting the stub from the generated code?

Reply:
the stub will call this func is my plan, to implement the sai API we need global functions, but those would be harder to test than class methods. So the stubs are going to call the methods on a global instance of the translator class